doc/admin/deployment.md

```diff
@@ -387,8 +387,8 @@ available at
 <https://github.com/daos-stack/daos/tree/master/utils/config>
 
 The location of this configuration file is determined by first checking
-for the path specified through the -o option of the daos\_server command
```

Comment:
Out of curiosity, why were the escapes added in the first place? It seems like we're flipping back and forth between escaped and unescaped underscores -- is this an artifact of the markdown editor being used?

Reply:
I think it was something to do with early version of the way @johannlombardi was rendering it, we are removing them now.